frontend/src/pages/pipelines/create/CreatePipeline.tsx

```diff
@@ -968,10 +968,7 @@ const Content: React.FC<ContentProps> = (props: ContentProps) => {
     } else {
       createPipelineObj.streaming = {
         appIdStreamList: pipelineInfo.streaming?.appIdStreamList ?? [],
-        retentionHours:
-          pipelineInfo.selectedStreamingDataRangeUnit?.value === 'day'
-            ? parseInt(pipelineInfo.streamingDataRangeValue) * 24
-            : parseInt(pipelineInfo.streamingDataRangeValue) || 1,
+        retentionHours: 24,
```

Comment:
I prefer to leavling this value as blank. Then updating the default value to `24` in template's parameter.

To avoid overriding the value after editing/upgrading the pipeline if the default is manually changed.